doc/man7/migration_guide.pod

```diff
@@ -467,6 +467,17 @@ Remove your usage of the low level APIs. In this case you will need to rewrite y
 
 =back
 
+=head3 Error code changes
+
+As OpenSSL 3.0 provides a brand new Encoder/Decoder mechanism for working with
+widely used file formats, the application authors should consider changes the checks
+making on parsing. The password-protected keys in PKCS#8/PKCS#12 formats deserve a
```

Comment:
I'd drop the `in PKCS#8/PKCS#12 formats` part.

Reply:
Not sure. It's a typical scenario where we get a lot of questions and regressions.

Reply:
The thing is it might happen also for the old password protected files in the PEM PKCS#1 encrypted format.

Reply:
Fixed. Thanks for the explanation!